solutions/CSES Fixed-Length Paths II.mdx

```diff
@@ -5,6 +5,8 @@ title: Fixed-Length Paths II
 author: Andi Qu
 ---
 
+Check out Benq's $\mathcal O(N)$ [solution](https://cses.fi/paste/871652b7df3be6d6189995/).
```

Comment:
Preferably add an explanation :)

Reply:
How do you show that it's linear?

Reply:
merging subtrees of depths a<=b takes O(a) time and results in subtree of depth b

Reply:
But then how do you show that O(sum of all depths - sum of largest depths in children) = O(N)? Is what I've said right now fine?